source/agora/consensus/EnrollmentManager.d

```diff
@@ -160,6 +160,9 @@ public class EnrollmentManager
         // create Pair object from KeyPair object
         this.key_pair = Pair.fromScalar(secretKeyToCurveScalar(key_pair.secret));
 
+        // load enrollment key
+        this.enroll_key = this.getEnrollmentKey();
+
```

Comment:
This value is never read. Currently a validator wishing to enroll will call the enrollment manager with the UTXO key. Do we need to read it here ?